README.md

```diff
@@ -1 +1,84 @@
-# Stock-Market
\ No newline at end of file
+# Stock-Market
+
+A web application that provides you with the data about the most active trading stocks for the day.
+
+## Deployment
+
+https://stocks-market-app.herokuapp.com/
+
```

Comment:
- [ ] Video presentation of your project is missing!  As part of this project, it is required you make a video presentation of your project which should include the following guideline, and as mentioned in the previous review:
- [ ] The maximum length of the video is 5 minutes.
- [ ] The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- [ ] The video contains a demo of the project, highlighting all the project features.
- [ ] The video contains a highlight about a piece of the code the student finds interesting or is proud of making.